plugins/inputs/solr/solr.go

```diff
@@ -0,0 +1,424 @@
+	SolrMbeans []json.RawMessage `json:"solr-mbeans"`
+}
+
+// ResponseHeader is an exported type that
+// contains a response metrics: QTime and Status
+type ResponseHeader struct {
+	QTime  int `json:"QTime"`
+	Status int `json:"status"`
+}
+
+// Core is an exported type that
+// contains Core metrics
+type Core struct {
+	Class string `json:"class"`
+	Stats struct {
+		DeletedDocs int `json:"deletedDocs"`
+		MaxDoc      int `json:"maxDoc"`
+		NumDocs     int `json:"numDocs"`
+	} `json:"stats"`
+}
+
+// QueryHandler is an exported type that
+// contains query handler metrics
+type QueryHandler struct {
+	Class string `json:"class"`
+	Stats struct {
+		One5minRateReqsPerSecond float64 `json:"15minRateReqsPerSecond"`
+		FiveMinRateReqsPerSecond float64 `json:"5minRateReqsPerSecond"`
+		Seven5thPcRequestTime    float64 `json:"75thPcRequestTime"`
+		Nine5thPcRequestTime     float64 `json:"95thPcRequestTime"`
+		Nine99thPcRequestTime    float64 `json:"999thPcRequestTime"`
+		Nine9thPcRequestTime     float64 `json:"99thPcRequestTime"`
+		AvgRequestsPerSecond     float64 `json:"avgRequestsPerSecond"`
+		AvgTimePerRequest        float64 `json:"avgTimePerRequest"`
+		Errors                   int     `json:"errors"`
+		HandlerStart             int     `json:"handlerStart"`
+		MedianRequestTime        float64 `json:"medianRequestTime"`
+		Requests                 int     `json:"requests"`
+		Timeouts                 int     `json:"timeouts"`
+		TotalTime                float64 `json:"totalTime"`
+	} `json:"stats"`
+}
+
+// UpdateHandler is an exported type that
+// contains update handler metrics
+type UpdateHandler struct {
+	Class string `json:"class"`
+	Stats struct {
+		Adds                     int    `json:"adds"`
+		AutocommitMaxDocs        int    `json:"autocommit maxDocs"`
+		AutocommitMaxTime        string `json:"autocommit maxTime"`
+		Autocommits              int    `json:"autocommits"`
+		Commits                  int    `json:"commits"`
+		CumulativeAdds           int    `json:"cumulative_adds"`
+		CumulativeDeletesByID    int    `json:"cumulative_deletesById"`
+		CumulativeDeletesByQuery int    `json:"cumulative_deletesByQuery"`
+		CumulativeErrors         int    `json:"cumulative_errors"`
+		DeletesByID              int    `json:"deletesById"`
+		DeletesByQuery           int    `json:"deletesByQuery"`
+		DocsPending              int    `json:"docsPending"`
+		Errors                   int    `json:"errors"`
+		ExpungeDeletes           int    `json:"expungeDeletes"`
+		Optimizes                int    `json:"optimizes"`
+		Rollbacks                int    `json:"rollbacks"`
+		SoftAutocommits          int    `json:"soft autocommits"`
+	} `json:"stats"`
+}
+
+// Cache is an exported type that
+// contains cache metrics
+type Cache struct {
+	Class string `json:"class"`
+	Stats struct {
+		CumulativeEvictions int     `json:"cumulative_evictions"`
+		CumulativeHitratio  float64 `json:"cumulative_hitratio,string"`
```

Comment:
I have SOLR 5.3.1 and could reproduce the issue. I found the values at:

http://[SERVER]:8983/solr/[CORE]/admin/mbeans?stats=true&wt=json&cat=CORE&cat=QUERYHANDLER&cat=UPDATEHANDLER&cat=CACHE

An extract of the whole JSON is:


`{
  "solr-mbeans": [
    {
      "filterCache": {
        "class": "org.apache.solr.search.FastLRUCache",
        "version": "1.0",
        "description": "Concurrent LRU Cache(maxSize=512, initialSize=512, minSize=460, acceptableSize=486, cleanupThread=false)",
        "src": null,
        "stats": {
          "lookups": 6,
          "hits": 0,
          "hitratio": 0,
          "inserts": 6,
          "evictions": 0,
          "size": 6,
          "warmupTime": 0,
          "cumulative_lookups": 6017884,
          "cumulative_hits": 9729,
          "cumulative_hitratio": 0,
          "cumulative_inserts": 6008155,
          "cumulative_evictions": 3110
        }
      }
    }
  ]
}`